README.md

```diff
@@ -0,0 +1,9 @@
+## Build
+
+```
+$ ./autogen.sh
```

Comment:
Those source distributions with the autogenerated files are created with `make dist` (which is how the release tarball should be created).